Message.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="us-ascii"?>
+<Message xmlns="http://schemas.microsoft.com/servicebusexplorer">
+  <Date>Friday, October 4, 2019 10:14:45 PM</Date>
+  <Content><![CDATA[<message>Hi mate, how are you?</message>]]></Content>
+</Message>
```

Comment:
What are these Message.xml and Properties.xml files?

Reply:
We do not need them, add to PR by mistake. I will remove them.